docs/src/main/sphinx/functions/table.rst

```diff
@@ -75,3 +74,38 @@ parameters in arguments::
     SELECT * FROM TABLE(my_function("row_count" => ? + 1, "column_count" => ?));
 
     EXECUTE stmt USING 100, 1;
+
+List of table functions by connector
```

Comment:
Not sure we want to do that .. so far we have not done so in any other SQL or functions or other generic docs .. instead we say it varies by connector ..

Reply:
I think it would be convenient to have them listed in one place. In the future, we will add built-in table functions, which will be listed here too.

Reply:
Okay ... just keep in mind that we have to continue to update this .. maybe even have a comment in the code as a reminder might help